### ISSUE TITLE
[BTC] BTC update requires long Utxos Fix

### ISSUE THREAD

**Issue by aman-rac** [trezor/blockbook#707]
Hello @martinboehm, we updated our blockbook and BTC Node to bitcoin core 22.0 and blockbook [0.3.5](https://github.com/trezor/blockbook/commit/6bbb81ec10fde2a38031285f7872c4123f1d513c). We got such logs for around 48 hours and the API and UI were down - 

`FixUtxos: row 560250432, addrDesc ad:76a914de158d14dd77665d1e599bb1ddb8078200e396d888ac, error balance 33852, checksum 34398, from txa 33852, txs 1076, fixed true`

After 48 hours the blockbook is working fine. Any way to prevent this? Thanks in advance!

**Comment by martinboehm** [comment]
Hi, about a year ago we discovered an issue with the Blockbook sync, which could in very infrequent times cause incorrect Utxo data to be stored in DB. We implemented a fix, which runs once after the the start of Blockbook. As you DB was already fixed, it will not need to run again and the next startup of Blockbook will be normal.
I am surprised a bit that it took so long time to fix the DB, in our cases it took only a couple of minutes.
